activity_browser/app/settings.py

```diff
@@ -2,16 +2,12 @@
 import os
 import json
 
-import appdirs
+from .. import PACKAGE_DIRECTORY
 
 
 class ABSettings():
     def __init__(self):
-        ab_dir = appdirs.AppDirs('ActivityBrowser', 'ActivityBrowser')
-        self.settings_dir = ab_dir.user_data_dir
-        if not os.path.isdir(self.settings_dir):
-            os.mkdir(self.settings_dir)
-        self.settings_file = os.path.join(self.settings_dir, 'ABsettings.json')
+        self.settings_file = os.path.join(PACKAGE_DIRECTORY, 'ABsettings.json')
```

Comment:
`PACKAGE_DIRECTORY` is an elegant way to access resources that we distribute with the code, eg the splash screen png. However I suspect we will run into problems if we use it to store the settings. It is not guaranteed that the user has write access to the `PACKAGE_DIRECTORY`.
- Example 1 (Windows): We create a stand-alone executable in the future for easy installation on windows. `PACKAGE_DIRECTORY` will be in `C:\Program Files\ActivityBrowser` or similar which is write-protected
- Example 2 (Windows or Linux): System wide install: If you install the AB to be used by several users, appdirs will store the settings separately for all users, however only admin/root would be able to write to `PACKAGE_DIRECTORY`

Reply:
But you are right that this raises the problem with several installs of the AB, ie they would all use the same settings file.